packages/kit/src/views/Staking/pages/ProtocolDetailsV2/index.tsx

```
@@ -527,66 +527,69 @@ function RiskSection({ risk }: { risk?: IStakeEarnDetail['risk'] }) {
+                    color="$iconSubdued"
+                    size="small"
+                    bg="transparent"
+                    onPress={() => {
+                      openUrlExternal(item?.actionButton?.data?.link);
+                    }}
+                  />
+                ) : null}
+              </XStack>
+
+              {item.list?.length ? (
+                <YStack gap="$1">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)